<?xml version="1.0"?>
<launch>
    <node pkg="turtlesim" type="turtlesim_node" name="turtlesim_node" output="screen" />
    <node pkg="turtlebot_controller" type="move_action_server" name="move_action_server" output="screen" />
    
</launch>
